CHANGELOG.md

```diff
@@ -12,7 +19,6 @@
 
 ## Documentation Updates
 - Added enabled config variables to `brand`, `organization` and `organization_tag` in the `src_zendesk.yml` models. 
-- [Updated README](https://github.com/fivetran/dbt_zendesk_source?tab=readme-ov-file#step-4-enabledisable-models-for-non-existent-sources) with instructions on how to disable `brand` and `organization` sources. 
```

Comment:
Could you add this back in?

Reply:
Ah thanks for catching. It must've been deleted during my recent fetch.